classes/Infrastructure/Event/ExceptionListener.php

```diff
@@ -46,40 +45,25 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         $exception = $event->getException();
         $request   = $event->getRequest();
 
-        $event->setResponse($this->renderResponse($request, $exception));
-    }
-
-    private function renderResponse(Request $request, \Throwable $exception): Response
-    {
-        if (\in_array('application/json', $request->getAcceptableContentTypes())) {
-            $headers = [];
-
-            if ($exception instanceof HttpExceptionInterface) {
-                $code    = $exception->getStatusCode();
-                $headers = $exception->getHeaders();
-            }
-
-            return new JsonResponse([
-                'error' => $exception->getMessage(),
-            ], $code, $headers);
+        if (!\in_array('application/json', $request->getAcceptableContentTypes())) {
+            return;
         }
 
-        $template = 'error/500.twig';
-
-        $templates = [
-            Response::HTTP_UNAUTHORIZED => 'error/401.twig',
-            Response::HTTP_FORBIDDEN    => 'error/403.twig',
-            Response::HTTP_NOT_FOUND    => 'error/404.twig',
-        ];
+        $event->setResponse($this->renderResponse($exception));
+    }
 
-        $code = $exception instanceof HttpExceptionInterface ? $exception->getStatusCode() : 500;
+    private function renderResponse(\Throwable $exception): Response
+    {
+        $code    = 500;
+        $headers = [];
 
-        if (\array_key_exists($code, $templates)) {
-            $template = $templates[$code];
+        if ($exception instanceof HttpExceptionInterface) {
+            $code    = $exception->getStatusCode();
+            $headers = $exception->getHeaders();
         }
 
-        $message = $this->twig->render($template);
```

Comment:
Since we aren't using `$twig` here anymore, can we remove the field as well as the constructor?

See https://github.com/opencfp/opencfp/blob/04086a1bea1eb16816291aae7137603dacd15806/classes/Infrastructure/Event/ExceptionListener.php#L27-L35.

Reply:
Good catch, I'm removing it.